src/components/Tutorial/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import Joyride from 'react-joyride'
+
+const Tutorial = props => {
+  const { history, steps, updateTutorialWatched } = props
+
+  const locale = {
+    back: 'Edellinen',
+    close: 'Sulje',
+    last: 'Valmis',
+    next: 'Seuraava',
+    skip: 'Ohita',
+  }
+
+  const customStyles = {
+    buttonNext: {
+      backgroundColor: '#f59023',
+      color: 'black',
+      fontFamily: 'Poppins',
+      fontSize: '16px',
+      borderRadius: '30px',
+      padding: '5px 10px',
+    },
+    buttonBack: {
+      marginLeft: 'auto',
+      marginRight: 5,
+
+      fontFamily: 'Poppins',
+      fontSize: '16px',
+      borderRadius: '30px',
+      padding: '5px 10px',
+      color: 'black',
+    },
```

Comment:
Adding buttonClose: {
      display: 'none',
    },
would dispose of the close button easily :)

Reply:
True, that could happen. I'm not sure I would update tutorialWatched if the user skips something since that might happen accidentally.. I think it might be ideal that if the user starts watching the tutorial, they could be forced to sit through the tutorial by preventing the modals from closing on outside click..  But the option to skip is good, I guess.